### PR TITLE
Add tabs

### PR DESCRIPTION
**Pull request by breezyboa**
Addresses #61

### CODE REVIEW

**Comment by breezyboa** [comment]
@sudeeptarlekar  _a please review

File: app/assets/stylesheets/client/app.scss [hunk 1]
```diff
@@ -171,10 +167,55 @@
...
+                    left: 40px;
+                }
+                &.highlight-library{
+                    left: 280px;
+                }
+                &.highlight-contactus{
+                    left: 520px;
+                }
+            }
+        }
```
Comment (sudeeptarlekar) [inline app/assets/stylesheets/client/app.scss]: @rishimohan Please review styles

**Comment by sudeeptarlekar** [comment]
Looks good to me. Once review from @rishimohan is done will merge PR.

**Comment by rishimohan** [comment]
LGTM

**Pull request merged**